src/components/TextEditor/components/TextEditor.js

```diff
@@ -199,136 +209,133 @@ class TextEditor extends React.Component {
    * @returns {string} the codemirror theme - see https://codemirror.net/demo/theme.html for more info
    */
 
-  getCMTheme = (theme) => {
-    switch (theme) {
-      case 'light':
-        return 'duotone-light';
-      case 'dark':
-      default:
-        return 'material';
+  const getCMTheme = (theme2) => {
```

Comment:
hm `theme2` is a bit arbitrary -- maybe we could use something like `newTheme` or `t`?